libs/perun/components/src/lib/services-status-list/services-status-list.component.html

```diff
@@ -0,0 +1,80 @@
+  <div class="card-body">
+    <perun-web-apps-table-options (exportEnd)="exporting = false" (exportStart)="exporting = true" [exporter]="exporter" class="ml-auto"></perun-web-apps-table-options>
+    <table mat-table matTableExporter [dataSource]="dataSource" #exporter="matTableExporter" matSort matSortActive="id" matSortDirection="asc" matSortDisableClear
+           class="w-100">
+
+      <ng-container matColumnDef="select">
+        <th mat-header-cell *matHeaderCellDef>
+          <mat-checkbox color="primary"
+                        (change)="$event ? masterToggle() : null"
+                        [checked]="selection.hasValue() && isAllSelected()"
+                        [indeterminate]="selection.hasValue() && !isAllSelected()"
+                        [aria-label]="checkboxLabel()">
+          </mat-checkbox>
+        </th>
+        <td mat-cell *matCellDef="let serviceState" class="static-column-size">
+          <mat-checkbox color="primary"
+                        (click)="$event.stopPropagation()"
+                        (change)="$event ? selection.toggle(serviceState) : null"
+                        [checked]="selection.isSelected(serviceState)"
+                        [aria-label]="checkboxLabel(serviceState)">
+          </mat-checkbox>
+        </td>
+      </ng-container>
+
+      <ng-container matColumnDef="id">
+        <th mat-header-cell *matHeaderCellDef
+            mat-sort-header>{{'SHARED_LIB.PERUN.COMPONENTS.SERVICES_STATUS_LIST.TASK_ID' | translate}}</th>
+        <td mat-cell class="static-column-size" *matCellDef="let serviceState">{{serviceState.task.id}}</td>
+      </ng-container>
+      <ng-container matColumnDef="serviceName">
+        <th mat-header-cell *matHeaderCellDef
+            mat-sort-header>{{'SHARED_LIB.PERUN.COMPONENTS.SERVICES_STATUS_LIST.SERVICE_NAME' | translate}}</th>
+        <td mat-cell *matCellDef="let serviceState">{{serviceState.service.name}}</td>
+      </ng-container>
+      <ng-container matColumnDef="status">
+        <th mat-header-cell *matHeaderCellDef>{{'SHARED_LIB.PERUN.COMPONENTS.SERVICES_STATUS_LIST.STATUS' | translate}}</th>
+        <td mat-cell *matCellDef="let serviceState">{{serviceState.task.status}}</td>
+      </ng-container>
+      <ng-container matColumnDef="blocked">
+        <th mat-header-cell *matHeaderCellDef>{{'SHARED_LIB.PERUN.COMPONENTS.SERVICES_STATUS_LIST.BLOCKED' | translate}}</th>
+        <td mat-cell *matCellDef="let serviceState">{{serviceState | serviceStateBlockedToString}}</td>
+      </ng-container>
+      <ng-container matColumnDef="started">
+        <th mat-header-cell *matHeaderCellDef
+            mat-sort-header>{{'SHARED_LIB.PERUN.COMPONENTS.SERVICES_STATUS_LIST.STARTED' | translate}}</th>
+        <td mat-cell *matCellDef="let serviceState">{{serviceState.task.startTime | millisecondsFromEpochToDate}}</td>
```

Comment:
you dont have to create you own pipe. Angular already provides you pipe for dates and various formats of showing them. For this case use pipe:
`date:'d.M.y h:mm:ss a'`